extensions/vscode/CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+# 🦄 Contributing to the Dart Frog Visual Studio Code extension
+
+First of all, thank you for taking the time to contribute! 🎉👍 Before you do, please carefully read this guide.
+
+## Opening an issue
+
+We highly recommend [creating an issue][bug_report_link] if you have found a bug, want to suggest a feature, or recommend a change. Please do not immediately open a pull request. Opening an issue first allows us to reach an agreement on a fix before you put significant effort into a pull request.
+
+When reporting a bug, please use the built-in [Bug Report][bug_report_link] template and provide as much information as possible including detailed reproduction steps. Once one of the package maintainers has reviewed the issue and we reach an agreement on the fix, open a pull request.
+
+## Developing for Dart Frog's Visual Studio Code extension
+
+To develop for the Dart Frog Visual Studio Code extension you will need to become familiar with VS Code extension development and the Very Good Ventures processes and conventions:
+
+### Setting up your local development environment
+
+1. Install a valid [Dart SDK](https://dart.dev/get-dart) in your local environment. If you have Flutter installed, you likely have a valid Dart SDK version already installed.
```

Comment:
"valid" is a broad term, we have version constraints so we should refer to those to define "valid"

Something like "Install a Dart SDK that is within the version constraints defined [here]"